rpc/core/blocks.go

```diff
@@ -76,8 +76,11 @@ func Block(ctx *rpctypes.Context, heightPtr *int64) (*ctypes.ResultBlock, error)
 		return nil, err
 	}
 
-	blockMeta := blockStore.LoadBlockMeta(height)
 	block := blockStore.LoadBlock(height)
```

Comment:
so there's a block, but no block meta? how's this possible?

Reply:
from my understanding this `return &ctypes.ResultBlock{BlockID: blockMeta.BlockID, Block: block}, nil` panics if there is no blockmeta to blockid

Reply:
but why is there no block meta?

Reply:
> I don't believe there is block meta for height 0. 

Right. There's no block either (for height 0), but in `LoadBlock` we first load `BlockMeta` and, if it's `nil`, return `nil` block.